### DISCUSSION TITLE
维护一个开源活动TimeLine项目

### DISCUSSION THREAD

**Discussion by xgdyp** [X-lab2017/open-wonderland#87] [2️⃣ 聊项目]
这几天我发现很多同学想参与到开源的一些暑期活动中，但是信息的收集成本比较高，有些活动可能没有了解到。我们可以发起一个开源项目，该项目维护了像ospp、gsoc等开源活动的时间节点。同学们可以通过关注我们的项目从而实现一站式了解开源活动，了解的多的同学也可以通过pr的形式来补充信息。大家怎么看？

**Comment by tyn1998** [comment]
那必须支持啊！

**Reply by xgdyp** [reply]
我们先给项目起个名字 有没有好的建议

**Reply by tyn1998** [reply]
我来一个：socgo

**Comment by TieWay59** [comment]
https://github.com/deepanshu1422/List-Of-Open-Source-Internships-Programs

**Reply by TieWay59** [reply]
This name is literally verbose but clear, everyone can google and find it easily in seconds. We can translate it into a Chinese version for convenience, I think. 😆

**Reply by xgdyp** [reply]
感叹两点：
果然能想到的东西基本都有人做过了
居然出了gsoc还有这么多活动

**Comment by TieWay59** [comment]
我现在也改口了，就叫开源实习，去搜 open source internship 可以搜到很多 list。比如：

- https://navendu.me/posts/open-source-internship-programs/
- https://iq.opengenus.org/open-source-internships/
- https://www.geeksforgeeks.org/best-openhttps://www.geeksforgeeks.org/best-open-source-programs-for-students-to-participate/-source-programs-for-students-to-participate/

**Comment by TieWay59** [comment]
如果时最近的活动的话，可以关注一下Github 的维护者月的日程，或者北美开源峰会（6月21日）：

https://maintainermonth.github.com/schedule

https://events.linuxfoundation.org/open-source-summit-north-america/

**Comment by TieWay59** [comment]
事情挺散的，这种list的价值可能在于有人足够频繁地去关注和维护。

比如 https://github.com/anubhavpulkit/Open-Source-Events 这个项目，就是大半年没维护了，那我其实也就不会觉得多好，我们完全可以取而代之。

**Comment by will-ww** [comment]
确实还是挺不错的，我们需要想想能够长期结合的点，这样能够促使我们不断地维护下去。

例如，持续的维护与更新，是否能和我们目前的项目、科研、教学等场景做结合，或者能帮我们采集到一些额外的数据、连接新的社区朋友等，或者发动其他开源社区一起来做等，都可以考虑下~

**Comment by xgdyp** [comment]
昨天和 @tyn1998 讨论了下整体的启动方式。我觉得烨男的想法很赞！我们可以联动 高校开源参与榜#117 ，通过榜单的形式对比各个学校的开源活动参与情况，可以发一篇类似于报告一样的推文，裂变过程中不了解SoC的读者会自然而然的想到哪里有这样的活动信息呢，顺带就推广了我们的这个项目～

**Reply by tyn1998** [reply]
明明都是翔宇你的idea，是我觉得你的想法好赞！
